frontend/src/lib/lemon-ui/LemonTable/LemonTable.scss

```diff
@@ -293,8 +293,10 @@
 
         .posthog-3000 & {
             &:hover {
-                .LemonTable__header-content {
-                    color: var(--default);
+                &:not(:has(.LemonTable__header--no-hover:hover)) {
```

Comment:
Reads kind of complicated but not too bad if you reason through it. Would it be enough to say `&:not(:has(.LemonTable__header--no-hover))` or is the extra `:hover` qualifier needed?

Reply:
Thanks. The `:hover` qualifier is required to scope the change to hover effects only (and other allow colors when not hovering).